### PR TITLE
adding keras layers for dealing with masks in variable length sequence inputs

### PR DESCRIPTION
**Pull request by iskandr**

### CODE REVIEW

File: mhcflurry/keras_layers/drop_mask.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from keras.layers import Layer
+
+class DropMask(Layer):
```
Comment (timodonnell) [inline mhcflurry/keras_layers/drop_mask.py]: Add a header comment about what this is?

File: mhcflurry/parallelism.py [hunk 2]
```diff
@@ -1,5 +1,6 @@
-from concurrent import futures
 import logging
+from six import PY2
```
Comment (timodonnell) [inline mhcflurry/parallelism.py]: why the import?

**Comment by timodonnell** [comment]
LGTM, made two minor comments
